Source/Core/DolphinQt2/Settings/GeneralPane.cpp

```diff
@@ -38,6 +38,8 @@ void GeneralPane::CreateLayout()
   CreateAnalytics();
 #endif
   CreateAdvanced();
+
+  m_main_layout->setMargin(0);
```

Comment:
Repeating what I said on IRC for posterity:

This margins change doesn't do anything to vertical margins for me (macOS). It only gets rid of horizontal margins, which makes the window look too squished :/